pilot/pkg/networking/grpcgen/grpcgen_test.go

```diff
@@ -26,12 +25,16 @@ import (
 	"google.golang.org/grpc/resolver"
 	"google.golang.org/grpc/serviceconfig"
 
+	// To setup the env vars needed for grpc-go. Should be loaded before grpc/xds is loaded.
```

Comment:
is golang initialization order based on import order or something?

Reply:
it is based on import ordering